### PR TITLE
Board Size: Use Gdk to get screen size

### PR DESCRIPTION
**Pull request by tombettany**
When retrieving the screen size to be able to correctly set the game
board size, Kano Settings is utilised. This fails because reading the
settings has become read-protected and so the whole game fails to
launch. Bypass this problem by reading the screen size directly from
Gdk.

cc @Ealdwulf @radujipa @pazdera

### CODE REVIEW

File: snake/stage.py [hunk 1]
```diff
@@ -13,9 +13,10 @@
 import __main__
 import os
 import time
+import gdk
 
 from kano.window import _get_window_by_child_pid, gdk_window_settings
-from kano_settings.system.display import get_status
+from kano.logging import logger
```
Comment (Ealdwulf) [inline snake/stage.py]: Kano.logging takes a second to import, you might want to import it where you generate an error message.

**Comment by Ealdwulf** [comment]
Apart from a minor comment, looks good. :+1:

**Comment by radujipa** [comment]
Could you please check if kano-settings is now used anywhere else in Make Snake? If not, it might be worth removing the dependency and bumping the version.

**Comment by Ealdwulf** [comment]
Actually, looks like it is also used in `snake-editor/stage.py`

**Comment by radujipa** [comment]
:shipit:

**Comment by tombettany** [comment]
@Ealdwulf Fixed the one in `snake-editor/stage.py`

**Comment by Ealdwulf** [comment]
Looks good :+

**Comment by pazdera** [comment]
I'm a bit late for the party, but this is using gtk2 right? Is there a reason for that?

**Comment by pazdera** [comment]
It should probably also depend on gtk when we're using it.

**Comment by tombettany** [comment]
@pazdera Just because we import `gdk_window_settings` which uses gtk2 so I kept this consistent to ensure that there are no conflicts.

**Comment by pazdera** [comment]
@tombettany Right, of course ... Didn't realise that.

**Comment by tombettany** [comment]
@pazdera You are probably right about the dependency, I'll add it in

**Pull request merged**